spec/apps/invalid-load-function/invalid-load-function.spec.js

```diff
@@ -46,14 +46,14 @@ export default function() {
           return Promise.resolve();
         }
       }
-      singleSpa.registerApplication('./invalid-load-function.app.js', loadFunction, location => location.hash === "#invalid-load-function");
+      singleSpa.registerApplication('./invalid-load-function.app.js 2', loadFunction, location => location.hash === "#invalid-load-function");
```

Comment:
After fixing #165 this file caused several issues. I just incremented the names for a quick fix